src/components/GlobalFilter/global-filter-menu.scss

```diff
@@ -1,7 +1,7 @@
 .chr-c-global-filter {
   padding: var(--pf-t--global--spacer--md) var(--pf-t--global--spacer--lg);
   background: var(--pf-t--global--background--color--100);
-  .pf-v6-c-chip-group { margin-right: var(--pf-t--global--spacer--xs); }
+  .pf-v6-c-label-group { margin-right: var(--pf-t--global--spacer--xs); }
```

Comment:
@Hyperkid123 I wonder if the codemods should have picked this up? Or were we using the wrong class prior to PF6?

Change looks fine to me otherwise.

Reply:
No clue why this was missed.